packages/apps/desktop-viewer-test/src/frontend/components/modelSelector/BriefcaseStatus.tsx

```diff
@@ -70,10 +67,10 @@ export const BriefcaseStatus = ({
         allClassNames += " actionable";
         break;
     }
+    return [ITwinViewerApp.translate(titleKey), allClassNames];
```

Comment:
id personally prefer returning an obj instead of an array
makes it easier as you add more to the return and i think cleaner when you destructure